apps/nextjs-website/src/components/organisms/WebinarQuestionsTable/WebinarQuestionsTable.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import WebinarQuestionRow from '@/components/molecules/WebinarQuestion/WebinarQuestionRow';
+import { updateWebinarQuestion } from '@/lib/webinarApi';
+import { useTranslations } from 'next-intl';
+import { WebinarQuestion } from '@/lib/webinars/webinarQuestions';
+
+type WebinarQuestionsTableProps = {
+  userName: string;
+  questions: WebinarQuestion[];
+  title: string;
```

Comment:
Following another comment made, if I see this property name, I assume this component receive the title to use for the table